temba/channels/views.py

```diff
@@ -527,6 +527,7 @@ def channel_status_processor(request):
         cutoff = timezone.now() - timedelta(hours=1)
         send_channel = org.get_send_channel(scheme=TEL_SCHEME)
         call_channel = org.get_call_channel()
+        ussd_channel = org.get_ussd_channel(scheme=TEL_SCHEME)
```

Comment:
Why do we have scheme for `get_ussd_channel` anyways? Is there any scenario where that can't be `TEL`?